private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -722,7 +722,7 @@ export class TemplateServiceClient {
     if (templateId && templates.length > 0) {
       await this._updateRecentTemplate(templateId, true);
 
-      if (!isClient) {
+      if (isClient === false) {
```

Comment:
why the check is like this? if it is undefined, you don't want to go in?

Reply:
Fixed, added check if undefined as well!